js/packages/web/src/views/pack/components/PackSidebar/index.tsx

```diff
@@ -15,12 +15,13 @@ interface IPropsPackSidebar {
 }
 
 const PackSidebar = ({ onOpenPack }: IPropsPackSidebar) => {
-  const { pack, voucherMetadataKey } = usePack();
+  const { pack, voucherMetadataKey, cardsRedeemed } = usePack();
```

Comment:
I would export **provingProcess** as well

```suggestion
  const { pack, voucherMetadataKey, provingProcess } = usePack();
```